core/kazoo/src/kz_network_utils.erl

```diff
@@ -123,6 +125,25 @@ is_cidr(Address) ->
         'error':'invalid_cidr' -> 'false'
     end.
 
+-spec get_local_ip() -> [].
+get_local_ip() ->
+{ok, Descriptions} = inet:getifaddrs(),
+IPs = lists:map(fun(Description) ->
```

Comment:
You can write the whole functions (and below function) for getting interface IP addresses like this:

```erlang
get_interfaces_ip() ->
    case inet:getifadrrs() of
        {'ok', IFs} ->
            [IP || {_, IFOpts} <- IFs,
                   {'addr', IP} <- IFOpts,
                   {'flags', Flags} <- IFOpts,
                   'up' <- Flags
            ];                                                                                                                             
        {'error', _Reason} ->
            lager:debug("failed to get interfaces: ~p", [_Reason]),
            []
    end.
```